### PR TITLE
Add overflow of main content for smaller screens

### PR DESCRIPTION
**Pull request by Winner95**
Now on screens less than 1200px content will be on top of sidebar.

Also added feature to resize content block of components

### CODE REVIEW

File: src/components/content/content.scss [hunk 1]
```diff
@@ -1,4 +1,7 @@
 .styleguide-content {
     padding: 32px;
     position: relative;
+    min-width: calc(900px - 17px); // 17px - scrollbar width;
```
Comment (kdzuin) [inline src/components/content/content.scss]: breaks UX to develop and test mobile components.
and there is unnecessary magic. can be done easier

File: src/components/sandbox/sandbox.scss [hunk 2]
```diff
@@ -44,10 +44,12 @@
 }
 
 .styleguide-sandbox__content {
+    overflow: auto;
     padding: 20px;
     border: 1px solid #f0f0f0;
     border-top: 0;
     transform: translate3d(0, 0, 0);
+    resize: both;
```
Comment (kdzuin) [inline src/components/sandbox/sandbox.scss]: breaks VRT by default

File: src/components/section/section.scss [hunk 3]
```diff
@@ -7,7 +7,7 @@
 }
 
 .styleguide-section__component {
-    max-width: 960px;
+    max-width: 1200px; // defines the width of component
```
Comment (kdzuin) [inline src/components/section/section.scss]: breaks VRT

File: src/components/content/content.scss (hunk 1 above)
Comment (Winner95) [reply src/components/content/content.scss]: @kdzuin updated diff as we agreed

File: src/components/section/section.scss (hunk 3 above)
Comment (Winner95) [reply src/components/section/section.scss]: Fixed

**Pull request merged**